packages/framework/fluid-framework/README.md

```diff
@@ -1,8 +1,29 @@
 # fluid-framework
 
-The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a corresponding service client package (ex. `@fluidframework/azure-client` & `@fluidframework/tinylicious-client`).
+The `fluid-framework` package bundles a collection of Fluid Framework client libraries for easy use when paired with a corresponding service client package (e.g. `@fluidframework/azure-client`, `@fluidframework/tinylicious-client`, or `@fluid-experimental/osdp-client (BETA)`).
 
-<!-- AUTO-GENERATED-CONTENT:START (README_DEPENDENCY_GUIDELINES_SECTION:includeHeading=TRUE) -->
+## Contents
+
+The `fluid-framework` package consists primarily of two portions: the `IFluidContainer` and a selection of distributed data structures (DDSes).
+
+### IFluidContainer
+
+The **[IFluidContainer][]** interface is the one of the types returned by calls to `createContainer()` and `getContainer()` on the service clients such as `AzureClient`.
+It includes functionality to retrieve the Fluid data contained within, as well as to inspect the state of the collaboration session connection.
+
+### DDS packages
+
+You'll use one or more DDS data structures in your container to model your collaborative data. The `fluid-framework` package comes with three data structures that cover a broad range of scenarios:
+
+1. **[SharedTree][]**
+1. **[SharedMap][]**, a map-like data structure for storing key/value pair data
+    - Note: as of version 2.0, `SharedMap` is deprecated. Please use `SharedTree` instead.
```

Comment:
Says "three", lists two. Also, is SharedMap actually deprecated? Or should we say something like "the recommended DDS for all new applications is SharedTree". Maybe even with "unless your collaborative data model is very simple" or something along those lines. Not sure pushing users to SharedTree when they just want a set of key-value pairs is ideal.

Reply:
SharedMap is actually marked as `@deprecated`, so this is in line with our guidance.

I'll update the out-of-date phrasing though.